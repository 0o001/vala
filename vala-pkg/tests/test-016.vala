using GLib;

class Maman.Bar {
	static int main (string[] args) {
		stdout.printf ("Conditional Expression Test: 1");
		
		stdout.printf (" %d", false ? -1 : 2);
		
		stdout.printf (" 3\n");
		
		return 0;
	}
}
