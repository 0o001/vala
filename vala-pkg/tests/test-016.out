Conditional Expression Test: 1 2 3
